department-of-veterans-affairs/vets-website
# DT a11y 508 header being read twice

## Description
A continuation of [15042](https://github.com/department-of-veterans-affairs/vets-website/pull/15042).
I misinterpreted the answer to the question of using paragraph tags with nested definition tag or h4 tags and making the `From Name:` syntax visible between Design and a11y teams. Here is the remedy. 

## Testing done
1. yarn build:content --pull-drupal   
2. yarn watch --env.entry=static-pages  
3. Navigate to => http://localhost:3001/find-forms/about-form-21p-0969/
4. Run a screen reader on the page to read the whole page. (`control + option + a`)
5. Notice the reader does not repeat the header.

## Acceptance criteria
- [x] The screen reader does not repeat the headers of the forms.

## Definition of done
